File: app/src/main/java/com/limelight/widget/GameListWidgetService.java
package com.limelight.widget;

import android.content.Intent;
import android.widget.RemoteViewsService;

public class GameListWidgetService extends RemoteViewsService {
    @Override
    public RemoteViewsFactory onGetViewFactory(Intent intent) {
        return new GameListRemoteViewsFactory(this.getApplicationContext(), intent);
    }
}

File: app/src/main/java/com/limelight/widget/GameListWidgetService.kt
package com.limelight.widget

import android.content.Intent
import android.widget.RemoteViewsService

class GameListWidgetService : RemoteViewsService() {
    override fun onGetViewFactory(intent: Intent): RemoteViewsFactory {
        return GameListRemoteViewsFactory(applicationContext, intent)
    }
}
